addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -219,3 +219,9 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
         }
     }
 }
+
+.form-select {
+    // This is necessary to achieve a consistent "border rendering behaviour"
+    // across form-select and form-control, particularly when using semi-transparent borders.
```

Comment:
Nitpicking:
- Max 80 characters per line for comments (and use those 80 characters of course)
- Commit title: `frontend inputs, maintain border consistency` => `maintain border consistency of frontend inputs`

Also remind me: what's the justification for this to be part of `bootstrap_review_frontend.scss` instead of `bootstrap_review.scss`. I think I failed to see the exact point when we split... or at least I forgot now. The point of the file was to "extend bootstrap to fix it / change its behavior in Odoo". Why would not we want to fix it the same way in backend and frontend? Feel like anything that should be frontend or backend only should not be in a file that contains the word "bootstrap".

Reply:
> Max 80 characters per line for comments (and use those 80 characters of course)

Done ✅ 

> Commit title: frontend inputs, maintain border consistency => maintain border consistency of frontend inputs

In the end, I've updated the branch using a single commit.

> Also remind me: what's the justification for this to be part of bootstrap_review_frontend.scss instead of bootstrap_review.scss

This fix is related to inputs with a semi-transparent border color and a background color.
If I'm not mistaken, this type of inputs is only defined in the frontend 🤔 .

I don't think we have inputs with a semi-transparent border color in the backend, or maybe I've missed something?
This is why it would make more sense to define it in `bootstrap_review_frontend.scss` rather than `bootstrap_review.scss`.

What do you think ?

cc. @stefanorigano

Reply:
Hello 👋

@qsm-odoo , @Brieuc-brd 

> I think I failed to see the exact point when we split... or at least I forgot now. The point of the file was to "extend bootstrap to fix it / change its behavior in Odoo".

The split has been introduced in https://github.com/odoo/odoo/pull/116641 (see in the PR description, SCSS section).

In short, there are "different Odoo(s)". The webclient deviated too much from BS (just see [how](https://github.com/odoo/odoo/blob/17.0/addons/web/static/src/scss/bootstrap_review_backend.scss) we render buttons there 😅) , while the frontend must remains "BS compliant" of course.
The split is needed to finetune the behaviour for a bundle (like the frontend input border-color) while avoiding to affect of the other one (see the complex rendering for webclient buttons, definitely not needed in the frontend).

Regarding the case of disabled inputs in the webclient:
- The current approach leans towards the pragmatic "_disabled inputs are unsightly, just hide them_" (sigh 😒).
- In isolated scenarios where Bootstrap inputs are utilized (like in knowledge, mail, POS, or rooms), the colors are solid, no need for this custo